### ISSUE TITLE
Add a note about "read-only" to combobox pattern

### ISSUE THREAD

**Issue by carmacleod** [w3c/aria-practices#1338]
Background discussion in https://github.com/w3c/aria-practices/issues/1026#issuecomment-586803681.

Some possible words:

> NOTE
> Additional authoring considerations for using `aria-readonly` and `aria-disabled`:
> - Limit use of `aria-readonly` to `combobox` widgets that have a text field. A read-only combobox is in the tab sequence and its value can be read, however its popup cannot be opened, and its value cannot be changed.
> - If a select-only combobox widget (i.e. a `combobox` without a text field) is not operable, use `aria-disabled="true"` to convey the disabled state to assisstive technologies.

The note could go right before the examples. @mcking65 please review.
Question: Should we say anything about a disabled read-only combobox?  :)

**Comment by JAWS-test** [comment]
> Should we say anything about a disabled read-only combobox?

A disabled red-only combobox should be the same as a simple disabled combobox. When I use disabled and read-only on an input element, read-only is ignored (see for example in Chrome, F12, Tab "Accessibility")
